templates/firefox-ios.xliff

```diff
@@ -3175,6 +3179,22 @@ Private Tab</source>
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.4" build-num="15F31d"/>
     </header>
     <body>
+      <trans-unit id="Alerts.AddToCalendar.Body.v134" xml:space="preserve">
+        <source>%@ is asking to download a file and add an event to your calendar.</source>
+        <note>When tapping on a link, on a website in order to download a file and that file is a calendar file, an alert comes up asking to confirm if you want to add the event to the device calendar. This is the body message for the alert. %@ is the name/domain of the website, for example 'google.com'</note>
+      </trans-unit>
+      <trans-unit id="Alerts.AddToCalendar.BodyDefaultDomain.v134" xml:space="preserve">
+        <source>This site</source>
```

Comment:
@adudenamedruby @dicarobinho
Flagged this in https://github.com/mozilla-mobile/firefox-ios/pull/23232#issuecomment-2488960898